CODEOWNERS

```diff
@@ -73,6 +73,9 @@ homeassistant/components/arest/* @fabaff
 homeassistant/components/asuswrt/device_tracker.py @kennedyshead
 homeassistant/components/automatic/device_tracker.py @armills
 homeassistant/components/axis/* @kane610
+homeassistant/components/*/arest.py @fabaff
+homeassistant/components/*/axis.py @kane610
```

Comment:
Why did you add this line? Merge issue?

Reply:
(Same as previous)
Yes indeed, I saw a couple of merge conflicts that I had to fix and I think I did something wrong there. It should not be part of this PR. Any idea how I can get this fixed? Should I just remove this line from the file and create a new commit?